src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, **kwargs):
         from azure.cli.core.commands.arm import (
             register_ids_argument, register_global_subscription_argument)
         from azure.cli.core.cloud import get_active_cloud
-        from azure.cli.core.extensions import register_extensions
+        from azure.cli.core.commands.transform import register as register_transform
```

Comment:
I would just change the method name to `register_global_transforms` and import it directly. It is more self-documenting.

Reply:
will do